command/csi.go

```diff
@@ -0,0 +1,73 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/mitchellh/cli"
+)
+
+type CSICommand struct {
+	Meta
+}
+
+func (f *CSICommand) Help() string {
+	helpText := `
+Usage: nomad csi [plugin|volume] <subcommand> [options] [args]
+
+  This command groups subcommands for interacting with CSI (Container Storage
+  Interface) Plugins and Volumes. CSI volumes provide network storage devices
+  to jobs running in containers. Healthy running plugins are necessary to use
+  volumes, plugins are configured in jobs running on the cluster. For a full
+  guide see https://www.nomadproject.io/guides/csi.html
+
+  Examine a plugin's status:
+
+      $ nomad csi plugin status <id>
+
+  List existing plugins:
+
+      $ nomad csi plugin list
+
+  Examine a volume's status:
+
+      $ nomad csi volume status <id>
```

Comment:
I realize this is an issue with having a PR before the RFCs have been fully reviewed, but having volumes under the top-level `nomad csi` seems like it cuts out a lot of design space for having command line operations on host volumes (which was left out of the original host volumes spec).